### ISSUE TITLE
MSCOCO Background

### ISSUE THREAD

**Issue by GloryyrolG** [mks0601/I2L-MeshNet_RELEASE#118]
Hi @mks0601 ,

May I ask where I can find "images of MSCOCO dataset that do not include humans to augment the backgrounds following Moon et al. [30]" to augment single-person 3DHP? Thanks & regards,

**Comment by mks0601** [comment]
Go to repo of that paper and you can download it

**Comment by GloryyrolG** [comment]
Hi, thanks for your reply. Did you mean this one [MuCo-3DHP parsed and composited data](https://drive.google.com/drive/folders/1dfhFa1kBHYKLTKuprNc7xixt3yyKEky5?usp=sharing)? I only saw `augmented_set` and `unaugmented_set` inside. I am wondering if there are background images and preprocessing code available. Regards,

**Comment by mks0601** [comment]
Yes I meant that. Unfortunately, I do not have masks and codes now..
